contracts/test/v0.1/RequestResponseConsumerContract.test.ts

```diff
@@ -146,10 +146,446 @@ describe('Request-Response user contract', function () {
+
+    // PREPAYMENT EVENT
+    const prepaymentEvent = prepaymentContract.interface.parseLog(fulfillReceipt.events[0])
+    expect(prepaymentEvent.name).to.be.equal('AccountBalanceDecreased')
+    expect(prepaymentEvent.args.accId).to.be.equal(accId)
+
+    // FIXME
+    // expect(prepaymentEvent.args.oldBalance).to.be.equal()
+    // expect(prepaymentEvent.args.newBalance).to.be.equal()
+
+    // COORDINATOR EVENT
+    const fulfillEvent = coordinatorContract.interface.parseLog(fulfillReceipt.events[1])
+    expect(fulfillEvent.name).to.be.equal('DataRequestFulfilledBytes32')
+    expect(fulfillEvent.args.requestId).to.be.equal(requestId)
+    expect(await consumerContract.s_responseBytes32()).to.be.equal(response)
+  })
+
+  it('Request & Fulfill Bytes', async function () {
+    const {
+      accId,
+      maxGasLimit,
+      consumerContract,
+      coordinatorContract,
+      prepaymentContract,
+      consumer,
+      rrOracle0
+    } = await loadFixture(deployFixture)
+    const prepaymentContractConsumerSigner = await ethers.getContractAt(
+      'Prepayment',
+      prepaymentContract.address,
+      consumer
+    )
+    const value = parseKlay(1)
+    await prepaymentContractConsumerSigner.deposit(accId, { value })
+    const requestReceipt = await (
+      await consumerContract.requestData(accId, maxGasLimit, {
+        gasLimit: 500_000
+      })
+    ).wait()
+
+    expect(requestReceipt.events.length).to.be.equal(1)
+    const requestEvent = coordinatorContract.interface.parseLog(requestReceipt.events[0])
+    expect(requestEvent.name).to.be.equal('DataRequested')
+
+    const eventArgs = [
+      'requestId',
+      'jobId',
+      'accId',
+      'callbackGasLimit',
+      'sender',
+      'isDirectPayment',
+      'data'
+    ]
+    for (const arg of eventArgs) {
+      expect(requestEvent.args[arg]).to.not.be.undefined
+    }
+
+    const { requestId } = requestEvent.args
+
+    const response = ethers.utils.formatBytes32String('hello')
+    const requestCommitment = {
+      blockNum: requestReceipt.blockNumber,
+      accId,
+      callbackGasLimit: maxGasLimit,
+      sender: consumerContract.address
+    }
+    const isDirectPayment = false
+
+    const coordinatorContractOracleSigner = await ethers.getContractAt(
+      'RequestResponseCoordinator',
+      coordinatorContract.address,
+      rrOracle0
+    )
+
+    const fulfillReceipt = await (
+      await coordinatorContractOracleSigner.fulfillDataRequestBytes(
+        requestId,
+        response,
+        requestCommitment,
+        isDirectPayment,
+        {
+          gasLimit: maxGasLimit + 300_000
+        }
+      )
+    ).wait()
+
+    expect(fulfillReceipt.events.length).to.be.equal(2)
+
+    // PREPAYMENT EVENT
+    const prepaymentEvent = prepaymentContract.interface.parseLog(fulfillReceipt.events[0])
+    expect(prepaymentEvent.name).to.be.equal('AccountBalanceDecreased')
+    expect(prepaymentEvent.args.accId).to.be.equal(accId)
+
+    // FIXME
```

Comment:
Do we need to keep this here? It repeats in every test.

Reply:
Yes, we can just keep 1. I put it in all tests, just in case something weird happens with prepayment

Reply:
I think I am missing the point. Why do we need an unfinished commented test?

Reply:
aha! that's just code garbage. I'll clean it up.